pkg/ssh/key_test.go

```diff
@@ -18,17 +18,17 @@ import (
 	"path/filepath"
 	"testing"
 
-	"github.com/okteto/okteto/pkg/model"
+	"github.com/okteto/okteto/pkg/okteto/constants"
 )
 
 func TestKeyExists(t *testing.T) {
 	dir := t.TempDir()
 
 	defer func() {
-		os.Unsetenv(model.OktetoFolderEnvVar)
+		os.Unsetenv(constants.OktetoFolderEnvVar)
 	}()
 
-	os.Setenv(model.OktetoFolderEnvVar, dir)
+	os.Setenv(constants.OktetoFolderEnvVar, dir)
```

Comment:
regarding tests, could we use the t.SetEnv to change within the test and then perform the cleanup ? https://pkg.go.dev/testing#T.Setenv

Reply:
I'll address this one here because it's really quick, but I think they are outside the scope of the pr

Reply:
i was concern about all the tests that where changed and do have `os.SetEnv`. This PR is a big change, from my perspective this could be a change inside the scope of the PR as we want to have the most stable test possible backing up. If you feel this is not the scope, then let this unchanged and i will open an issue of technical debt to fix the setting of ENVs at the unit tests so the env is just changed for the test.

Reply:
opened https://github.com/okteto/okteto/issues/3179